ws/websocket.go

```diff
@@ -382,7 +390,8 @@ func (server *Server) readPump(ws *WebSocket) {
 	conn := ws.connection
 	defer func() {
 		_ = conn.Close()
-		ws.closeSignal <- true
```

Comment:
idiomatic go seems to use `chan struct{}` for signalling (though more to write ;))

Reply:
Good point, will do in a future PR (check comment below)